netlify.toml

```diff
@@ -0,0 +1,8 @@
+[context.production.environment]
+    HOST = 'https://api.ceqr.app'
```

Comment:
I made this up, but seems if we are serving these separately, we should likely move the api to diff host/subdomain.

Reply:
Might just be something like `ceqr-app.herokuapp.com`. Will confirm when we're ready to merge and deploy.